en/news/_posts/2021-11-09-ruby-3-1-0-preview1-released.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: news_post
+title: "Ruby 3.1.0 Preview 1 Released"
+author: "naruse"
+translator:
+date: 2021-09-25 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby {{ release.version }}.
+
+{% assign release = site.data.releases | where: "version", "3.1.0-preview1" | first %}
+
+
+## YJIT: New experimental in-process JIT compiler
+
+
+Ruby 3.1 merges YJIT, a new in-process JIT compiler developed by Shopify.
+
+Since [Ruby 2.6 introduced MJIT in 2018](https://www.ruby-lang.org/en/news/2018/12/25/ruby-2-6-0-released/), its performance greatly improved, and finaly [we achieved Ruby3x3 last year](https://www.ruby-lang.org/en/news/2020/12/25/ruby-3-0-0-released/). But even though Optcarrot has shown impressive speedups, the JIT hasn't benefited real world business applications.
```

Comment:
It looks like a typo.

```suggestion
Since [Ruby 2.6 introduced MJIT in 2018](https://www.ruby-lang.org/en/news/2018/12/25/ruby-2-6-0-released/), its performance greatly improved, and finally [we achieved Ruby3x3 last year](https://www.ruby-lang.org/en/news/2020/12/25/ruby-3-0-0-released/). But even though Optcarrot has shown impressive speedups, the JIT hasn't benefited real world business applications.
```